exercises/custom-set/example.py

```diff
@@ -0,0 +1,42 @@
+class CustomSet(object):
+    def __init__(self, elements=[]):
+        self.elements = list(elements)
+
+    def empty(self):
+        return not any(self.elements)
```

Comment:
This should be `return not self.elements` - the current version would wrongly return `True` if `self.elements == [0]`

Reply:
Good catch... fixing.